src/apps/MASS3DEA.hpp

```diff
@@ -0,0 +1,133 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-23, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+///
+/// Action of 3D mass matrix via partial assembly
+///
+/// Based on MFEM's/CEED algorithms.
+/// Reference implementation
+/// https://github.com/mfem/mfem/blob/master/fem/bilininteg_mass_ea.cpp#L142
```

Comment:
This link goes nowhere.

Reply:
Ah, they re-organized their repo. Just pushed a fix.